### PR TITLE
Misc Machinery related changes

### PR DESCRIPTION
**Pull request by Jalleo**
Couple of areas touched by my machinery changes. A few old ones deleted. A holodeck related item removed along with a check removed as its a unused item. Also removal of powersink.dm as its very outdated and potentially doesn't work.

Along with a move over to tool_act for the fusion generators

## Why It's Good For The Game

Less crud code

:cl: Jalleo
code: Code made slightly better in tiny areas. Report any issues about the fusion generator or elsewhere
/:cl:

### CODE REVIEW

**Comment by Jalleo** [comment]
I will test this PR later on locally I made it during a larger PR where I am touching multiple other files.

File: code/game/machinery/computer/guestpass.dm [hunk 1]
```diff
@@ -6,34 +6,14 @@
 	desc = "Allows temporary access to station areas."
```
Comment (DominikPanic) [inline code/game/machinery/computer/guestpass.dm]: Are the guest IDs used anywhere? If not, you can safely remove it entirely.

File: code/game/machinery/flasher.dm [hunk 2]
```diff
@@ -37,14 +37,13 @@
...
 //Don't want to render prison breaks impossible
-/obj/machinery/flasher/attackby(obj/item/W as obj, mob/user as mob)
-	if (iswirecutter(W))
-		add_fingerprint(user)
-		src.disable = !src.disable
-		if (src.disable)
-			user.visible_message("<span class='warning'> [user] has disconnected the [src]'s flashbulb!</span>", "<span class='warning'> You disconnect the [src]'s flashbulb!</span>")
-		if (!src.disable)
-			user.visible_message("<span class='warning'> [user] has connected the [src]'s flashbulb!</span>", "<span class='warning'> You connect the [src]'s flashbulb!</span>")
+/obj/machinery/flasher/wirecutter_act(mob/living/user, obj/item/W)
```
Comment (DominikPanic) [inline code/game/machinery/flasher.dm]: I assume this was supposed to be in your other PR

File: code/game/machinery/fusion_engine.dm [hunk 3]
```diff
@@ -145,97 +145,102 @@
...
-					return TRUE
-			else
-				to_chat(user, "<span class='warning'>You need more welding fuel to complete this task.</span>")
-				return FALSE
-	else if(iswirecutter(O))
-		if(buildstate == FUSION_ENGINE_MEDIUM_DAMAGE && !is_on)
+	else
+		return ..()
+
+/obj/machinery/power/fusion_engine/welder_act(mob/living/user, obj/item/O)
```
Comment (DominikPanic) [inline code/game/machinery/fusion_engine.dm]: Same here

File: code/game/machinery/computer/guestpass.dm (hunk 1 above)
Comment (Jalleo) [reply code/game/machinery/computer/guestpass.dm]: They are located in big red

File: code/game/machinery/fusion_engine.dm (hunk 3 above)
Comment (Jalleo) [reply code/game/machinery/fusion_engine.dm]: It was meant to be here. The first proper one is going to be larger in scale.

File: code/game/machinery/flasher.dm (hunk 2 above)
Comment (Jalleo) [reply code/game/machinery/flasher.dm]: This was a more generic misc related and was meant to be here.

Comment (DominikPanic) [reply code/game/machinery/flasher.dm]: Not really proper atomization but meh, it's a small PR

**Comment by Rohesie** [comment]
Merge conflict.

File: code/game/machinery/computer/guestpass.dm [hunk 4]
```diff
@@ -6,34 +6,14 @@
...
-	else
-		to_chat(usr, "<span class='notice'>This pass expires at [worldtime2text(expiration_time)].</span>")
+	to_chat(user, "This card looks like its been sitting here for some time.")
 
-	to_chat(usr, "<span class='notice'>It grants access to following areas:</span>")
-	for (var/A in temp_access)
-		to_chat(usr, "<span class='notice'>[get_access_desc(A)].</span>")
-	to_chat(usr, "<span class='notice'>Issuing reason: [reason].</span>")
-	return
+/obj/item/card/id/guest/read()
```
Comment (Rohesie) [inline code/game/machinery/computer/guestpass.dm]: If this is an override for a different proc, add a `return` to for clarity.

File: code/game/machinery/flasher.dm [hunk 5]
```diff
@@ -37,14 +37,13 @@
...
-	. = ..()
-	if(iswirecutter(I))
-		disable = !disable
-		if(disable)
-			user.visible_message("<span class='warning'> [user] has disconnected the [src]'s flashbulb!</span>", "<span class='warning'> You disconnect the [src]'s flashbulb!</span>")
-		else
-			user.visible_message("<span class='warning'> [user] has connected the [src]'s flashbulb!</span>", "<span class='warning'> You connect the [src]'s flashbulb!</span>")
+/obj/machinery/flasher/wirecutter_act(mob/living/user, obj/item/W)
+	add_fingerprint(user)
+	src.disable = !src.disable
```
Comment (Rohesie) [inline code/game/machinery/flasher.dm]:
```suggestion
	disable = !disable
```

File: code/game/machinery/flasher.dm [hunk 6]
```diff
@@ -37,14 +37,13 @@
...
+	if (src.disable)
```
Comment (Rohesie) [inline code/game/machinery/flasher.dm]:
```suggestion
	if (disable)
```

File: code/game/machinery/flasher.dm [hunk 7]
```diff
@@ -37,14 +37,13 @@
...
+		user.visible_message("<span class='warning'> [user] has disconnected the [src]'s flashbulb!</span>", "<span class='warning'> You disconnect the [src]'s flashbulb!</span>")
+	if (!src.disable)
```
Comment (Rohesie) [inline code/game/machinery/flasher.dm]:
```suggestion
	if (!disable)
```

**Pull request merged**